_plugins/prettify.rb

```diff
@@ -1,77 +1,112 @@
+# Copied from https://github.com/dart-lang/site-shared/blob/master/src/_plugins/prettify.rb
```

Comment:
@filiph - FYI, I've chosen, for now, to just copy the file rather than setup use of site-shared (I don't want to introduce too many changes upfront).